### ISSUE TITLE
exceptions when running examples

### ISSUE THREAD

**Issue by jthies** [Exawind/nalu-wind#1070]
Hi,
we're new to nalu-wind and trying to run an example or two to see how it works.
There may be obvious usage errors, in which case I apologize for opening an issue.
I followed the user guide [here](https://nalu-wind.readthedocs.io/_/downloads/en/latest/pdf).

I managed to compile everything, but in the simulation step with naluX, I get these exceptions:

## abl_neutral:'''
'''
  what():  TKE active with wall bc, however, no value of tke or wall function specified
'''

## turbine_uniform_inflow

 what():  Symmetry::fatal_error()

In the latter case, there was an outdated entry in a yaml file (.i had to be replaced by .yaml), so I suspect that these examples are not part of continuous integration testing. Any idea what might be going on? I checked out nalu-wind at revision ead59abf5 (master branch),
and openfast at 5f3fb6ef74f4. I had to patch the cmake system here and there but did not change the code or example files.

**Comment by tasmith4** [comment]
@psakievich do you know what's going on here or do you know who might?  It wouldn't surprise me at all if our examples were out-of-date ...

**Comment by psakievich** [comment]
Yes these are all out of date. We have had an issue for sometime to remove them see #836. Since we haven't been keeping them up to date.  That said, it might be worth adding this into our nightly test process to run the examples, but I'm not sure when we could get to it.  I think there is value in the examples cases but it takes consistent effort and reminders to keep them up to date.

I'm also not sure if the wind-utils repo is up-to-date either.

@jthies for CI tested input files please see: https://github.com/Exawind/nalu-wind/tree/master/reg_tests/test_files/ablNeutralEdge

The regression test files are kept up to date and tested nightly.  We apologize for the inconvenience, but you should be able to debug any diffs with this as a reference.

Also for help building you can checkout  the nalu-wind package in https://github.com/spack/spack
We are very heavily dependent on spack these days and it is our preferred method of deployment.

Another option is to pull down one of our docker containers https://hub.docker.com/r/ecpe4s/exawind-snapshot/tags.
A new image is pushed every night to docker hub.

**Comment by jthies** [comment]
Thanks for the suggestions!

- The problem with the second test case (turbine_uniform_inflow) is that mesh parts have different names in the different yaml files. It's running in principle, although I now run into another issue.
- I had a few minor issues with wind-utils and openfast, in particular, in both packages there's a line liike 
'''
link_libraries(yaml-cxx)
'''
missing.
- The docker containers may be interesting to try because we just want to run an example or two. Doesn't fix the problem that we know nothing about running these cases, though -- we actually work on improving the solvers in Trilinos that you use and want a "real world" example for a paper :)

Finally, I think the examples are great, especially in combination with the PDF, and I would really recommend running them automatically from time to time to keep them working. Things like renamed boundaries are easy to fix when you see a test fail after a commit.

**Comment by psakievich** [comment]
@jthies we will try to get someone to update the examples and automate them. We definitely see value in it. As with all things it's a matter of resources available right now. In terms of urgency, are you currently stuck or were you able to get the cases running?
